EncouragePackage/EncouragePackage.csproj

```diff
@@ -82,15 +82,6 @@
     <Reference Include="WindowsFormsIntegration" />
   </ItemGroup>
   <ItemGroup>
-    <COMReference Include="Microsoft.VisualStudio.CommandBars">
```

Comment:
Removed as blocking compiling on VS2019. Tested locally, doesn't appear to be needed.